src/workflow/runtime/WorkflowContext.ts

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { OrchestrationContext } from "@microsoft/durabletask-js";
+import { Task } from "@microsoft/durabletask-js/task/task";
+import { TInput } from "@microsoft/durabletask-js/types/input.type";
```

Comment:
We have already defined TInput and TOutput in this PR and those are the types that we should use here. We should not return DTF types to users as a result of callActivity etc.

Reply:
Thanks for catching this! Updated!